src/flows/context/flow.list.tsx

```diff
@@ -82,6 +82,7 @@ export function serialize(flow) {
       if (meta) {
         return {
           ...flow.data.byID[id],
+          id,
```

Comment:
the reason the visualizations weren't displaying after the initial query when the notebook is loaded is because we weren't persisting the pipe's ID. So on initial load, the pipe's ID is regenerated and the results from the query get lost because the old IDs no longer exist. Persisting the IDs solves this problem.